content/en/docs/Performance/_index.md

```diff
@@ -6,6 +6,14 @@ description: >
   Baker performance test cases
 ---
 
+Baker has been designed with high performance in mind. Baker core, the part of the code base
+which distributes records among components and ties them together, is very high-quality Go code.
```

Comment:
```suggestion
which distributes records among components and ties them together, is very high-quality Go code.  
```

Reply:
double space to add a newline

Reply:
i know but I don't want a new line here :-)

Reply:
ok